solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -128,7 +128,7 @@ public class Http2SolrClient extends SolrClient {
   private static final List<String> errPath = Arrays.asList("metadata", "error-class");
 
   private HttpClient httpClient;
-  private volatile Set<String> queryParams = Collections.emptySet();
+  protected volatile Set<String> urlParamNames = Set.of();
```

Comment:
I don't think this needs to be protected anymore (following latest changes)

Reply:
@epugh bumping this comment up for visibility

Reply:
thank you!